.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Scala Merge CI
+
+on:
+  push:
+    branches: ['2.*.x']
+  pull_request:   # TODO remove before merging PR
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build_and_test:
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - run: git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Note that we don't use olafurpg/setup-scala; it wouldn't buy us anything
+      # over setup-java. (We don't want csbt or xsbt; we prefer the standard
+      # sbt launch script, which comes preinstalled on Windows (and Ubuntu).)
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
```

Comment:
Oh, the new setup-java action now supports AdoptOpenJDK.  Nice!  I didn't catch that change on #9496.